pkg/tsdb/cloudmonitoring/cloudmonitoring.go

```diff
@@ -429,7 +429,11 @@ func buildFilterString(metricType string, filterParts []string) string {
 }
 
 func buildSLOFilterExpression(q sloQuery) string {
-	return fmt.Sprintf(`%s("projects/%s/services/%s/serviceLevelObjectives/%s")`, q.SelectorName, q.ProjectName, q.ServiceId, q.SloId)
+	if q.SelectorName == "select_slo_burn_rate" {
+		return fmt.Sprintf(`%s("projects/%s/services/%s/serviceLevelObjectives/%s", "%s")`, q.SelectorName, q.ProjectName, q.ServiceId, q.SloId, q.LookbackPeriod)
+	} else {
+		return fmt.Sprintf(`%s("projects/%s/services/%s/serviceLevelObjectives/%s")`, q.SelectorName, q.ProjectName, q.ServiceId, q.SloId)
+	}
```

Comment:
Since the format is a little bit complicated than that you commented and an example is `select_slo_burn_rate("projects/PROJECT_ID/services/SERVICE_ID/serviceLevelObjectives/SLO_ID", "3600s")`, I didn't apply the suggestion as it is. Also the LookbackPeriod parameter is sent even with other SelectorName when switching it from UI so only checking if LookbackPeriod is empty doesn't work. 
However, applied the DRY and modify the condition a bit here https://github.com/grafana/grafana/pull/53710/commits/dbdeaf189d1b9236fd753f9d61c6d57e160466a7